src/lib/travis.js

```diff
@@ -24,8 +24,8 @@ const travisyml = {
   node_js: ['9', '8', '6', '4'], // eslint-disable-line camelcase
   after_success: ['npm run travis-deploy-once "npm run semantic-release"'], // eslint-disable-line camelcase
   branches: {
-    // ignore git tags created by semantic-release, like "v1.2.3"
-    except: [/^v\d+\.\d+\.\d+$/.toString()],
+    // Avoid double build on PRs (see: https://github.com/travis-ci/travis-ci/issues/1147)
+    only: ['master'],
```

Comment:
Yeah, that's a fair point. I've always treated `semantic-release-cli`'s output as a starting point and make changes afterwards. I guess the point here is deciding what's considered "good defaults". `master` is the default release branch, so this would work _by default_.

On the other hand, maybe this is surprising and is better suited to a "travis tips" section in the docs. Either way, happy to err on the side of whatever's least of a burden for y'all :)

Reply:
Yeah maybe we could add tips to https://semantic-release.gitbooks.io/semantic-release/docs/recipes/travis.html and https://semantic-release.gitbooks.io/semantic-release/docs/recipes/travis-build-stages.html? 

I would probably add both `only: ['master'],` as well as the config at https://github.com/octokit/rest.js/blob/aa816a6ea086b0d6ea39ca77845c345c3151cf5a/.travis.yml#L7-L12 as many folks use semantic-release together with greenkeeper

Reply:
Both solution (`except : [^v\d+\.\d+\.\d+$]` and `only: [master]`) would create a problem when semantic-release/semantic-release#563 lands as we would release from multiple branches.
We had several debates regarding what to recommend in the cli and in the doc regarding "best practice". We decided to recommend the least possible as it creates a lot of (sometimes conflictual/passionate) debate as it seems to be a subject on which everyone has an opinion.

So I would rather not make any recommendation, not even in the doc.
What we can do though is to link blog article in [docs/resources.md](https://github.com/semantic-release/semantic-release/blob/caribou/docs/resources.md).

For this PR I think we should just remove `branch` altogether.